packages/edit-site/src/components/global-styles/context-menu.js

```diff
@@ -24,48 +41,97 @@ function ContextMenu( { name, parentMenu = '' } ) {
 	const hasLayoutPanel = hasDimensionsPanel;
 	const hasVariationsPanel = useHasVariationsPanel( name, parentMenu );
 
+	const { canEditCSS } = useSelect( ( select ) => {
```

Comment:
There is some code duplication here from [screen-root.js](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/global-styles/screen-root.js#L29), I could not figure out how or if I could pass the `canEditCSS` value from the screen root to the next panel via the `<NavigationButtonAsItem>` ?

Reply:
@carolinan, probably it's not worth it. We've hooks like `useSelect` to avoid prop drilling.